src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -447,6 +447,143 @@ JSON example:
+|  LabelingOrientation          | RECOMMENDED  | [array][] of [numbers][]    | Orientation of the labeling plane (`(P)CASL`) or slab (`PASL`). The direction cosines of a normal vector perpendicular to the ASL labeling slab or plane with respect to the patient. Corresponds to DICOM Tag 0018,9255 `ASL Slab Orientation`.|
+|  LabelingDistance    | RECOMMENDED  | [number][]  | Distance from the center of the imaging slab to the center of the labeling plane (`(P)CASL`) or the leading edge of the labeling slab (`PASL`), in millimeters. If the labeling is performed inferior to the isocenter, this number should be negative. Based on DICOM macro C.8.13.5.14. |
+|  LabelingLocationDescription     | RECOMMENDED   | [string][]    | Description of the location of the labeling plane (`(P)CASL`) or the labeling slab (`PASL`) that cannot be captured by fields ‘LabelingOrientation’ or ‘LabelingDistance’. May include a link to an anonymized screenshot of the planning of the labeling slab/plane with respect to the imaging slab or slices `sub-<label>[_ses-<label>][_acq-<label>][_rec-<label>][_run-<index>]_labeling.jpg`. Based on DICOM macro C.8.13.5.14.  |
+| LookLocker  | OPTIONAL | [boolean][]  | Boolean indicating if a Look-Locker readout is used.| 
+|  LabelingEfficiency     | OPTIONAL   | [number][]  | Labeling efficiency, specified as a number between zero and one. |
+
+### (P)CASL-specific metadata fields
+These fields can only be used when `LabelingType` is `CASL` or `PCASL`.
+| **Key name**                  | **Requirement level** | **Data type** | **Description**                                                   |
+|-------------------------------|-----------------------|---------------|------------------------------------------------------------------------------------------|
+|  LabelingDuration                 | REQUIRED            | [number][] or [array][] of [numbers][]         | Total duration of the labeling pulse train, in seconds, corresponding to the temporal width of the labeling bolus for `(P)CASL`. Specify either one number for total time-series or provide an array of numbers of different values for each volume in case of sequential acquisitions with varying labeling duration. In the case that an array of numbers is provided, its length should be equal to the number of volumes specified in  `sub-<label>[_ses-<label>][_acq-<label>][_rec-<label>][_run-<index>]_aslcontext.tsv`. An `m0scan` acquired within the ASL time-series, gets a value of zero. Corresponds to DICOM Tag 0018,9258 `ASL Pulse Train Duration`. |
```

Comment:
```suggestion
|  LabelingDuration                 | REQUIRED            | [number][] or [array][] of [numbers][]         | Total duration of the labeling pulse train, in seconds, corresponding to the temporal width of the labeling bolus for `(P)CASL`. Specify either one number for total time-series or provide an array of numbers of different values for each volume in case of sequential acquisitions with varying labeling duration. In the case that an array of numbers is provided, its length should be equal to the number of volumes specified in the associated `aslcontext.tsv`. An `m0scan` acquired within the ASL time-series, gets a value of zero. Corresponds to DICOM Tag 0018,9258 `ASL Pulse Train Duration`. |
```

This will be easier to read, unless there is a reason I'm missing to include the full name.

Reply:
It would be great to apply this to all of the instances of full filenames in these tables as well.

Reply:
done